src/tools/linkchecker/linkcheck.sh

```diff
@@ -55,18 +50,33 @@ do
         *)
             if [ -n "$book_name" ]
             then
-                echo "only one argument allowed"
-                exit 1
+                if [ -n "$book_path" ]
+                then
+                    echo "at most two arguments allowed"
```

Comment:
Since this is optional, can it maybe be a `--path` argument?  I think that would be roughly something like:

```sh
--path)
    book_path="$2"
    shift
    ;;
```